bin/crew

```diff
@@ -4,7 +4,14 @@ require 'digest/sha2'
 require 'json'
 require 'fileutils'
 require 'tmpdir'
-require_relative '../commands/const'
+begin
+  require_relative '../commands/const'
+rescue LoadError
+  # Older crew installs won't have the commands dir in the sparse checkout,
```

Comment:
Don't we already handle this in lib/fixup.rb?

Reply:
The error happens when loading crew, before we get to fixup.

Reply:
But how will we have updated to a newer state without running fixup?

Reply:
Git pull has already happened at this point. Disabling the sparse checkout unhides all the files needed.

Reply:
The git pull leaves crew in a broken state, because the commands dir is not yet part of the sparse checkout.